content/pt/docs/languages/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: APIs e SDKs de Linguagens
+description:
+  A instrumentação de código do OpenTelemetry é suportada para muitas linguagens
+  populares de programação.
+weight: 250
+redirects: [{ from: /docs/instrumentation/*, to: ':splat' }]
+default_lang_commit: b3a087c468094eb144ab3114aff06838c8fb06ca
+---
+
+A [instrumentação][] de código do OpenTelemetry é suportada para as linguagens
+listadas na tabela de [Estado e Lançamentos](#estado-e-lançamentos) abaixo.
```

Comment:
Do not translate heading anchors, see https://opentelemetry.io/docs/contributing/localization/#heading-anchors